styles.css

```diff
@@ -656,10 +791,141 @@ form {
     gap: 24px;
   }
 
+  .portfolio_techs {
+    grid-column: 1/3;
+    grid-row: 2/3;
+  }
+
+  .portfolio_description {
+    grid-column: 1/3;
+    grid-row: 3/4;
+  }
+
   .work_item {
     margin: 0;
   }
 
+  .modal {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    display: none;
+    overflow: scroll;
+    justify-content: center;
+    background: #c1c7d0;
+  }
```

Comment:
- [ ] As requested by the previous code reviewer, From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
  - On the desktop, your modals should have margin-top and margin-bottom to match the figma design

| Your Design |Figma Design |
| :---:   | :---: | 
| ![Screenshot from 2022-12-21 09-04-29](https://user-images.githubusercontent.com/92303891/208865041-e3d1b211-a4cc-46e2-9ef1-c953194e32cd.png)| ![image](https://user-images.githubusercontent.com/92303891/208863113-77c4f0a0-7146-403e-a53d-cbcecd2d1dca.png)|